_extensions/bluesky-comments/bluesky-comments.js

```diff
@@ -131,6 +132,12 @@ class BlueskyComments extends HTMLElement {
     return null;
   }
 
+  #convertToHttpUrl(uri) {
+    uri = this.#convertToAtProtoUri(uri);
+    const [, , did, , rkey] = uri.split('/');
+    return `https://bsky.app/profile/${did}/post/${rkey}`;
```

Comment:
Currently hard-wired to `https:/bsky.app` as the base URL, but this gives us a place to allow for user configured settings